### ISSUE TITLE
Support for AngularJS 1.7 LTS

### ISSUE THREAD

**Issue by matedo1** [patternfly/angular-patternfly#766]
At this moment angular-patternfly supports [1] only Angular 1.5.*

It would be very useful to have the support also for AngularJS 1.7 LTS that will be supported until June 30, 2021 [2] 

Thanks

[1] https://github.com/patternfly/angular-patternfly/blob/master/bower.json#L40
[2] https://blog.angular.io/stable-angularjs-and-long-term-support-7e077635ee9c

**Comment by thescouser89** [comment]
Any update on this issue?

**Comment by dtaylor113** [comment]
Hi @matedo1, unfortunately angular-patternfly is in maintenance mode, meaning we would only implement fixes for P1 bugs.   However, if you were to open a PR which added support for AngularJS 1.7, we would give it the highest priority for review.  Thanks.

**Comment by dtaylor113** [comment]
Hi @matedo1, thanks to the great work of @redallen, we have updated angular-patternfly to angular 1.7.7.   This is a major bump for angular-patternfly, so you will need to use angular-patternfly@5.0.0.   

There are no known breaking changes, however there where a lot of source code changes and dependency version updates, so we decided to minimize risks to existing customers who don't necessarily want all those changes and don't necessarily need angular 1.7.7.   Thanks!
